packages/orchestration/src/orchestration-response.ts

```diff
@@ -51,6 +55,56 @@ export class OrchestrationResponse {
     return choice?.message?.content;
   }
 
+  /**
+   * Messages that can be used for subsequent prompts as message history.
+   * @param choiceIndex - The index of the choice to parse.
+   * @returns A list of all messages.
+   */
+  getAllMessages(choiceIndex = 0): ChatMessages {
+    const messages: ChatMessage[] = (
+      this.data.module_results.templating ?? []
+    ).map(message =>
+      this.isMultiChatMessage(message)
+        ? this.handleMultiChatMessage(message as MultiChatMessage)
```

Comment:
[req] You dont need an assertion here now, since the type guard takes care of that.